docs/worker/handlers/on_stop.md

```diff
@@ -0,0 +1,18 @@
+# on_stop
+
+The `on_stop` handler is invoked when a task called the `stop` method or a when a SIGTERM signal received.
```

Comment:
The `on_stop` handler is invoked when a task calls the `stop` method or receives a `SIGTERM` signal.